gtsfm/runner/gtsfm_runner_base.py

```diff
@@ -30,6 +32,7 @@
 logger = logger_utils.get_logger()
 
 DEFAULT_OUTPUT_ROOT = Path(__file__).resolve().parent.parent.parent
+REACT_METRICS_PATH = Path(__file__).resolve().parent.parent.parent / "rtf_vis_tool" / "src" / "images"
```

Comment:
that doesnt look right. why is react metrics pointing to a different directory now?

Reply:
changes the name to REACT_IMAGES_PATH in a new commit. The path is different but it should not be in the same folder as the metrics in my opinion